src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -0,0 +1,208 @@
+  VA_FORM_IDS.VIC,
+];
+
+const configs = [
+  fullSchema1010ez,
+  // Remap the formId to match the name in vets-json-schema
+  // This should only affect the mapping in the "check all forms" test
+  { ...fullSchema526AllClaims, formId: '21-526EZ-ALLCLAIMS' },
+  fullSchema686,
+  fullSchema527EZ,
+  fullSchema530,
+  fullSchema0993,
+  fullSchema0994,
+  fullSchema1990,
+  fullSchema1990e,
+  fullSchema1990n,
+  fullSchema1995,
+  fullSchema5490,
+  fullSchema5495,
+  fullSchema10007,
+  fullSchemaFeedbackTool,
+  fullSchemaVIC,
+];
+
+// These forms do not have formConfig but are found in vets-json-schema/dist/schemas
+const excludedForms = new Set([
+  '28-1900',
+  '28-8832',
+  '24-0296',
+  VA_FORM_IDS.FORM_22_1995_STEM, // unused
+  VA_FORM_IDS.FORM_21_526EZ, // old
+  'definitions',
+  'constants',
+  'vaMedicalFacilities',
+]);
+
+describe('form:', () => {
+  it('should check all forms', () => {
+    const allFormIds = Object.keys(schemas).filter(
+      formId => !excludedForms.has(formId),
+    );
+    const reformattedIds = mappedIds.slice(0);
+    reformattedIds.splice(0, 1, VA_FORM_IDS.FORM_10_10EZ);
+    const includedFormIds = configs.map(form => form.formId);
+
+    const allFormIdsSet = new Set(allFormIds);
+    const mappedIdsSet = new Set(mappedIds);
+    expect(allFormIdsSet.size).to.not.lessThan(
+      mappedIdsSet.size,
+      'a schema may have been removed from vets-json-schema/dist/schemas',
+    );
+    expect(allFormIdsSet.size).to.not.greaterThan(
+      mappedIdsSet.size,
+      'a schema may have been added to vets-json-schema/dist/schemas',
+    );
+    expect(allFormIdsSet).to.deep.equal(mappedIdsSet);
+    expect(includedFormIds).to.deep.equal(
+      reformattedIds,
+      'possible missing formId property in a formConfig',
+    );
+  });
+
+  configs.forEach(form => {
+    describe(`${form.formId}:`, () => {
+      describe('migrations:', () => {
+        const { migrations } = form;
+        if (migrations || form.version > 0) {
+          it('should have a length equal to the version number', () => {
+            expect(migrations.length).to.equal(form.version);
+          });
+          it('should be typeof array', () => {
+            expect(migrations).to.be.an('array');
+          });
+          it('should be array of functions', () => {
+            expect(
+              migrations.every(migration => typeof migration === 'function'),
+            ).to.be.true;
+          });
+        }
+      });
+
+      it('should have chapters object', () => {
+        expect(form.chapters).to.be.an('object');
+      });
+
+      it('should have defaultDefinitions object', () => {
+        expect(form.defaultDefinitions).to.be.an('object');
+      });
+
+      it('should have introduction function', () => {
+        expect(form.introduction).to.be.an('function');
+      });
+
+      it('should have prefillEnabled boolean', () => {
+        expect(form.prefillEnabled).to.be.a('boolean');
```

Comment:
Is this one necessary? If they don't pass this in, it'll be treated as `false` and won't break anything, right?

Reply:
will make it an optional test only if property exists